cou/apps/base.py

```diff
@@ -323,6 +343,17 @@ def current_os_release(self) -> OpenStackRelease:
             f"{', '.join(mismatched_repr)}. This is not currently handled."
         )
 
+    @property
+    def is_versionless(self) -> bool:
```

Comment:
Well, we can check if the charm name is inside `VERSIONLESS` instead of checking the version of the units.

Reply:
I don't think `not all(unit.workload_version for unit in self.status.units.values())` will ever get run because `is_charm_supported` will return false if a charm is in neither openstack lookup csv or VERSIONLESS list, which means no application class will be instantiated for it. Is that right? If so, I'd prefer only checks  `self.charm in VERSIONLESS` here

Reply:
After yesterday discussion, I think the right solution would be to create specific class for `glance-simplestreams-sync`, since this is only charm needing this and that's why we should not move this logic to the base class (`OpenStackApplicaion`)